Cue/app/tabs/library/LibraryHome.js

```diff
@@ -125,6 +126,44 @@ class LibraryHome extends React.Component {
   }
 
   _onPressAddDeck = () => {
+    Alert.alert(
+      Platform.OS === 'android' ? 'Add deck' : 'Add Deck',
+      'You can create a new private deck or add a shared deck by entering a share code.',
+      [
+        {text: 'Cancel', style: 'cancel'},
+        {text: 'Look Up Share Code', onPress: this._onPressLookUpDeckByShareCode},
+        {text: 'Create New Deck', onPress: this._onPressCreateDeck},
```

Comment:
Theres still that issue with IOS/Android rendering buttons in different orders right?